packages/core/src/utils/utils.ts

```diff
@@ -249,9 +249,13 @@ export const applyVariations =
  *
  * @public
  */
-export const getPaletteColor =
-  (...args) =>
-  (props) => {
+export function getPaletteColor(
+  ...args: [
+    PaletteColor | PaletteFamilyName | PaletteFamilyVariation | string,
+    (PaletteColor | PaletteFamilyVariation)?
+  ]
```

Comment:
Forcing a tuple type to get some intellisense support. Currently using `| string` for the first argument does not provide any intellisense help. But there is chatter that in future versions of TypeScript it will b e supported